### DISCUSSION TITLE
Can't actived copilot in VS 2022, login success but nothing happend

### DISCUSSION THREAD

**Discussion by li-zheng-hao** [community/community#19321] [Copilot Conversations]
![image](https://user-images.githubusercontent.com/22129824/175441759-8f31c801-52ec-4175-95b3-b518359d360c.png)

I'm in China, and i open Clash for windows (opened TUN mode) and can access github 、google or other website which is banned by GFW, but i can't login in copilot , here is the log info in vs 2022 console:
![image](https://user-images.githubusercontent.com/22129824/175442136-9af76cff-df36-460b-8d99-44576054a77b.png)

**Comment by mashirozx** [comment]
same with clash and vscode, can confirm network is ok but got 403 code
<img width="1158" alt="image" src="https://user-images.githubusercontent.com/16148054/175447546-55127602-3d09-41b2-b159-c14e7179df6b.png">

**Reply by li-zheng-hao** [reply]
now i login successed, i restart my system and open vs again, login copilot again, everything works ok

**Reply by mashirozx** [reply]
> now i login successed, i restart my system and open vs again, login copilot again, everything works ok

not work again after reload vscode window
